apps/web/src/components/docs/docs.const.ts

```diff
@@ -0,0 +1,19 @@
+import { ROUTES } from '@novu/shared-web';
+
+export const PATHS: { [key in ROUTES]?: string } = {
+  [ROUTES.ACTIVITIES]: 'activity-feed/introduction',
+  [`${ROUTES.BRAND}/layouts`]: 'content-creation-design/layouts',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** This manual template literal here is a bit of a code smell. Can we instead add the `ROUTES.LAYOUT` route into the `ROUTES` constant? After we make this change, we should be able to simplify the typing of `PATHS` to remove the `?` symbol.‏

Reply:
this will only live until the information architecture project is released and all the if cases are gone and this one can be removed.